src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Abstractions/Channel/TagEditor.cs

```diff
@@ -0,0 +1,96 @@
+﻿/*
```

Comment:
This file is a move and edit of `AirshipBindings.NETStandard.Abstractions/Push/TagEditor.cs`. Not sure why GitHub doesn't recognize it as that.